src/test/java/uk/gov/hmcts/reform/bulkscanccdeventhandler/validators/OcrDataValidatorTest.java

```diff
@@ -48,6 +48,49 @@ void should_return_errors_when_mandatory_fields_are_missing() {
             );
     }
 
+    @Test
+    void should_return_errors_when_duplicate_keys_exist() {
+        // given
+        List<OcrDataField> ocrDataFields = asList(
+            new OcrDataField(FIRST_NAME, "name"),
+            new OcrDataField(LAST_NAME, "xyz"), //duplicate key
+            new OcrDataField(LAST_NAME, "abc"), //duplicate key
+            new OcrDataField(DATE_OF_BIRTH, "01-01-1990")
+        );
+
+        // when
+        OcrValidationResult result = validator.validate(PERSONAL, ocrDataFields);
+
+        // then
+        assertThat(result).isNotNull();
+        assertThat(result.status).isEqualTo(ERRORS);
+        assertThat(result.warnings).isEmpty();
+        assertThat(result.errors)
+            .isNotEmpty()
```

Comment:
I guess something like `containsExactly...` would be enough for this `assertThat...`